### DISCUSSION TITLE
What's the difference between appregistry.registercomponent and export?

### DISCUSSION THREAD

**Discussion by JulianHSC** [microsoft/react-native-windows#12947] [Q&A] [answered]
I'm checking the internet. appregistry.registercomponent seems to be for multiple items as in multiple imports but export is only for one. Is this correct?

**Comment by JulianHSC** [comment, marked as answer]
I will use the basic export for now.

**Reply by chrisglein** [reply]
More information on JS exports: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export
And on React Native's AppRegistry: https://reactnative.dev/docs/appregistry
But unless you have a reason to do more... it sounds like you've picked the right defaults.

**Comment by chrisglein** [comment]
Can you provide links to the information your consulting, or sample code that you have tried?

**Comment by JulianHSC** [comment]
You can see the hello world in the training website: https://reactnative.dev/docs/tutorial. As for appregistry.registercomponent, I find that in other examples like how to enable different fonts, alignment of text, how to create buttons etc.
